src/main/resources/i18n/displayStrings.properties

```diff
@@ -2155,7 +2156,7 @@ payment.f2f.info='Face to Face' trades have different rules and come with differ
   happened at the meeting. In such cases the BTC funds might get locked up forever or until the trading peers come to \
   an agreement.\n\n\
   To be sure you fully understand the differences with 'Face to Face' trades please read the instructions and \
-  recommendations at: 'https://docs.bisq.network/#f2f'
+  recommendations at: 'https://docs.bisq.network/trading-rules.html#f2f-trading'
```

Comment:
(Optional) Maybe the URL could be removed from the property, since it's not being translated? Then we could append it to the translated property string in code.